### DISCUSSION TITLE
Query logs from Log Explorer outside of the Dashboard

### DISCUSSION THREAD

**Discussion by arkhodakov** [supabase/supabase#16608] [Questions] [answered]
Can't find any way to query platform logs from an external application.
I'm building a monitoring pipeline with Grafana and I would like to be aware of every error and warning that arise in Edge Functions logs. Is there a way to fetch these logs?

**Comment by GaryAustin1** [comment, marked as answer]
I believe this response says no and no plans near term.
https://github.com/orgs/supabase/discussions/14952#discussioncomment-6733506
